nistats/tests/test_glm_reporter.py

```
@@ -0,0 +1,319 @@
+import nibabel as nib
+import numpy as np
+
+from nose import SkipTest
+
+import pandas as pd
+from nilearn.datasets import fetch_oasis_vbm
+
+from nistats.design_matrix import make_first_level_design_matrix
+from nistats.first_level_model import FirstLevelModel
+from nistats.reporting import glm_reporter as glmr
+from nistats import datasets
```

use fake datasets to avoid spurious test failures